tests/test_e2e/test_relations.py

```diff
@@ -303,3 +303,10 @@ def post_event(charm: CharmBase):
 def test_cannot_instantiate_relationbase():
     with pytest.raises(RuntimeError):
         RelationBase("")
+
+
+def test_relation_ids():
+    initial_id = RelationBase.NEXT_RELATION_ID
```

Comment:
It seems odd to have a mutating counter in all caps.
That being said, now that it is auto incremented by subsequent calls to `Relation`, I guess we don't really need it to be public anymore..?

Reply:
it was always meant to auto-increment on Relation calls, only somehow it didn't?

Reply:
I thought your use case was: I want to predict what Id the next relation I instantiate will get, without having to call `relation.relation_id` after instantiating it

Reply:
I thought I needed it because I assumed that by design rel_id wasn't auto incrementing.